Kbs.Business/Boat/BoatEntity.cs

```diff
@@ -10,4 +10,5 @@ public class BoatEntity
     [Column("BoatTypeID")]
     public int BoatTypeId { get; set; }
     public BoatStatus Status { get; set; }
+    public DateTime DeleteRequestDate { get; set; }
```

Comment:
Maak deze nullable met een vraagteken, en vervang de checks met MinValue:
```suggestion
    public DateTime? DeleteRequestDate { get; set; }
```